src/service/fbs/node.fbs

```diff
@@ -91,6 +91,12 @@ table IngestTask {
   size: uint64;
   state: byte;
   date: uint64;
+  macrosAndValues: [MacroAndValue];
+}
+
+table MacroAndValue {
```

Comment:
Can we remove "And" by just `MacroValue`?  We can go with `KeyValue` too if that is more generic =